conda_recipe_manager/parser/recipe_reader.py

```diff
@@ -97,10 +97,10 @@ def _v1_sub_jinja() -> None:
         # then we fall back to performing JINJA substitutions.
         try:
             try:
-                output = cast(JsonType, yaml.safe_load(s))
+                output = yaml.load(s, Loader=yaml.CSafeLoader)
```

Comment:
You might want to do something like
```python
try:
    from yaml import CSafeLoader as SafeLoader
except ImportError:
    from yaml import SafeLoader
```
to support cases where the C bindings are not available.